src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { User, Todo } from '../../react-app-env';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  usersFromServer: User[],
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo, usersFromServer }) => {
```

Comment:
Can these be called "users"?

```suggestion
export const TodoInfo: React.FC<Props> = ({ todo, users }) => {
```